### ISSUE TITLE
Even more easier setup for Rails 

### ISSUE THREAD

**Issue by Willianvdv** [rmosolgo/graphql-ruby#914]
I'd like to make the setup for Rails users super simple.
- [ ] register Rake tasks automatically
- [ ] think about a default, but configurable, schema name.
- [ ] write a 'graphql-ruby' is still just Rails blogpost

**Comment by rmosolgo** [comment]
🤘

**Comment by jjoos** [comment]
Some experiences while working on #911 (I'm a n00b so please double check my assumptions)

The tl;dr; getting started on: http://graphql-ruby.org/ doesn't mention rake tasks, while http://graphql-ruby.org/getting_started does. (and the readme erroneously doesn't link to the getting started but to the home page) It wasn't clear for me what the generator exactly did since they generate an object and not a type. (I read the docs: http://graphql-ruby.org/schema/generators  and then ran `rails g graphql:object -h`).

It would be awesome if the generator would be able to do type inference from an existing model or also generate an ActiveRecord model.

It would be nice if there was a default rake task to print the schema (and optionally watch for file changes). I'm assuming that most implementation need this?

In a default rails app you need to add `skip_before_action :verify_authenticity_token` to use the endpoint from any client/curl. Maybe some documentation about this would be nice?

I was stuck for some time on the schema printer not writing a `schema` entry, but this was due to Query being the default name and the printer omitting it. (this was confusing and I though this was causing an unrelated error).

Overall the experience was pretty good, most of my issues where related to relay modern.

**Comment by rmosolgo** [comment]
Feel free to reopen this or another issue if you get a chance to try some of these items!
